salt/_pillar/metalk8s_endpoints.py

```diff
@@ -0,0 +1,57 @@
+"""Store data about bootstrap services ip/port in pillar"""
+
+import logging
+
+
+log = logging.getLogger(__name__)
+
+__virtualenv__ = 'metalk8s_endpoints'
```

Comment:
It's `__virtualname__`, not `__virtualenv__`

Reply:
oups, will fix